schemas/groups/navigation.json

```diff
@@ -36,153 +38,188 @@
+          "description": "The position of the last waypoint in 3 dimensions",
+          "$ref": "../definitions.json#/definitions/position"
         },
-
-        "source": {
-          "$ref": "../definitions.json#/definitions/source",
-          "description": "Source of last update to active route"
+        "nextWaypoint": {
+          "description": "The position of the next waypoint in 3 dimensions",
+          "$ref": "../definitions.json#/definitions/position"
         },
-
-        "bearingActual": {
-          "type": "number",
-          "description": "The current bearing of the next waypoint relative to true North",
+        "geometry": {
+          "title": "LineString",
+          "description": "The GeoJSON LineString that defines the course between the last and next waypoints. When null, use a straight line",
+          "allOf": [{
+                "$ref": "../definitions.json#/definitions/commonValueFields"
+            }],
+            "properties":{
+            "type": {
+              "enum": [
+                "LineString"
+              ]
+            },
+            "coordinates": {
+              "$ref": "../definitions.json#/definitions/lineString"
+            }
+          }
+        },
+        "nextMark": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "Array position (zero-based) of next mark in the geometry LineString"
+        },
+        "lastMark": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "Array position (zero-based) of last mark in the geometry LineString"
+        },
+        "vmg": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "The velocity made good towards the next waypoint",
+          "units": "m/s"
+        },
+        "courseRequiredTrue": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "The course relative to true North towards the next waypoint",
           "units": "rad"
         },
-
-        "distanceActual": {
-          "type": "number",
-          "description": "The current distance to the next waypoint",
+        "courseRequiredMagnetic": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "The course relative to magnetic North towards the next waypoint",
+          "units": "rad"
+        },
+        "timeToGo": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "Time to Go in seconds before we reach the next waypoint",
+          "units": "s"
+        },
+        "xte": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "Cross track error",
           "units": "m"
         },
-
-        "bearingDirect": {
-          "type": "number",
-          "description": "The bearing relative to true North from last waypoint to the next waypoint",
+        "eta": {
```

Comment:
We have mostly everything in verbose format, I think this and xte above should be written out as well.